eclair-core/src/main/scala/fr/acinq/eclair/io/Monitoring.scala

```diff
@@ -21,8 +21,10 @@ import kamon.Kamon
 object Monitoring {
 
   object Metrics {
-    val PeerConnections = Kamon.counter("peers.connecting.count")
-    val ConnectedPeers = Kamon.gauge("peers.connected").withoutTags()
+    val PeerConnectionConnecting = Kamon.counter("peerconnection.connecting")
+    val PeerConnectionConnected = Kamon.gauge("peerconnection.connected")
+
+    val PeerConnected = Kamon.gauge("peer.connected")
```

Comment:
Why did you change the name from `peers.connected` to `peer.connected`? Plural feels better here IMO

Reply:
> Plural feels better here IMO

Agree, it was to maintain consistency with `PeerConnectionConnected`.

Another idea I had was to use inner objects:

```scala
object PeerConnection {
      val Connecting = Kamon.counter("peerconnection.connecting")
      val Connected = Kamon.gauge("peerconnection.connected")
    }

    object Peer {
      val Connected = Kamon.gauge("peer.connected")
    }
```

But I refrained from doing that because you didn't use that pattern for other metrics.

Reply:
I think the name of the objects are fine, it's just the name of the metric (`peerconnection.*` and `peer.*`) where I would have used plural (`peerconnections.*` and `peers.*`). Not a big issue though.